devito/ir/iet/visitors.py

```diff
@@ -480,7 +480,7 @@ def visit_Expression(self, o):
             code = c.Assign(lhs, rhs)
 
         if o.pragmas:
```

Comment:
These four lines are repeated twice. Worth constructing a utility function?

Reply:
which four lines? I only see two, starting at `if o.pragmas` -- which is too little to deserve a separate function

Reply:
```
if o.pragmas:
            code = c.Module(self._visit(o.pragmas) + (code,))

return code
```
but tbh, you're probably right